backoffice/templates/backoffice/home.html

```diff
@@ -3,6 +3,24 @@
 {% block heading %}Dashboard{% endblock %}
 
 {% block main %}
+  {% if drivers_pending > 0 or accounts_pending > 0 %}
+    <a href="/backoffice/approvals/" class="bg-yellow-300 flex-row hover:bg-yellow-100 m-2 p-2">
+      ❗
+      {% if accounts_pending > 0 %}
+        <div class="inline-block">
+          {% if accounts_pending > 1 %}
+            {{accounts_pending}} billing accounts require approval
+          {% else %}
+            1 billing account requires approval
+          {% endif %}
+        </div>
+      {% endif %}
+      {% if drivers_pending > 0 %}
+        <div class="inline-block">{{drivers_pending}} driver requires approval</div>
```

Comment:
You've done the singular/plural separation for billing accounts but not for drivers. Can you do both consistently?

Reply:
Oh yes, well spotted, this was the code I wrote outside PZ train station. Excuses excuses.